zulipterminal/ui_tools/views.py

```diff
@@ -814,28 +845,18 @@ def __init__(self, controller: Any) -> None:
         max_widths = [max(width) for width in zip(*widths)]
         self.width = sum(max_widths)
 
-        help_menu_content = []  # type: List[urwid.AttrWrap]
+        help_menu_content = []
         for category in HELP_CATEGORIES:
-            if len(help_menu_content) > 0:  # Separate categories by newline
-                help_menu_content.append(urwid.Text(''))
-
-            help_menu_content.append(urwid.Text(('popup_category',
-                                                 HELP_CATEGORIES[category])))
-
             keys_in_category = (binding for binding in KEY_BINDINGS.values()
                                 if binding['key_category'] == category)
-            for help_item_number, binding in enumerate(keys_in_category):
-                help_menu_content.append(
-                    urwid.AttrWrap(
-                        urwid.Columns([
-                            urwid.Text(binding['help_text']),
-                            (max_widths[1],
-                                urwid.Text(", ".join(binding['keys'])))
-                            ], dividechars=2),
-                        None if help_item_number % 2 else 'popup_contrast'
-                    )
-                )
-
+            key_bindings = []
+            for binding in keys_in_category:
+                key_bindings.append((binding['help_text'],
+                                     ', '.join(binding['keys'])))
+            help_menu_content.append((HELP_CATEGORIES[category], key_bindings))
+
+        help_menu_content = self.make_table_with_categories(help_menu_content,
```

Comment:
This reassigning is a bit confusing; it's perfectly valid python, and strictly `Sequence[...]` is compatible with `List[Any]` (can you see where I got those from?), but just using a different name would probably make this clearer :)